### DISCUSSION TITLE
Recyclarr script still running if set to false

### DISCUSSION THREAD

**Discussion by quadcom** [RandomNinjaAtk/docker-radarr-extended#44] [General]
I filed this for Sonarr-Extended

https://github.com/RandomNinjaAtk/docker-sonarr-extended/issues/26

But it's also happening here. I run both Sonarr and Radarr locally but use a remote seedbox so I have to sync files to a local DIR and manually move them to be processed by Radarr and Sonarr respectively. I disabled both Recyclarr and Queue Cleaner but they are now running anyway.

![image](https://user-images.githubusercontent.com/7085931/226681487-672aff37-31c8-464c-8204-7d4e4e588e92.png)

![image](https://user-images.githubusercontent.com/7085931/226680942-d3caacfa-8f1d-4149-a0a1-808770c7ea5e.png)

**Comment by RandomNinjaAtk** [comment]
What is the script version? Make sure your on the latest....

Also review log output and provide log data to show that it is running....

**Comment by quadcom** [comment]
I can see the QueueCLeaner script is running.

![image](https://user-images.githubusercontent.com/7085931/226688676-135c7ac4-196d-4c17-94a3-1cbcad4d294f.png)

I'm unsure how to check the version of scripts, as everything is just in the image DL'd from DH from within Unraid. I understood that updating the container would update everything within the container. Are you saying that we have to run a CLI command to manage the updates to those scripts independently from updating the containers?

**Reply by RandomNinjaAtk** [reply]
You need to make sure you've checked the container for updates and run all updates. If you don't have a log file for the script, then the script is not running...

**Comment by quadcom** [comment]
Ok, so I'm not sure how this happened. 

I've never had Queuecleaner running ever since I discovered what it did -which broke my process. How this script was re-enabled, escapes me. 

Forcing an update on the container and restarting it after clearing the log files seems to have cleared things up. I don't see the QueueCleaner.log file getting created again.

At least for now.
